beeline-core/src/main/java/io/honeycomb/beeline/tracing/propagation/Propagation.java

```diff
@@ -22,4 +22,13 @@ private Propagation() {
     public static PropagationCodec<String> honeycombHeaderV1() {
         return HttpHeaderV1PropagationCodec.getInstance();
     }
+
+    /**
+     * Codec that can decode/encode trace context based on the Amazon X-Ray http header ('X-Amzn-Trace-Id').
```

Comment:
One nit, because we don't support the entire range of functionality in X-Ray, we're opting to just call these "Amazon AWS http headers". That reinforces that the use case is primarily supporting load balancer resources.